wrappers/python/examples/export_ply_example.py

```diff
@@ -18,7 +18,7 @@
 pipe = rs.pipeline()
 config = rs.config()
 # Enable depth stream
-config.enable_stream(rs.stream.depth)
+config.enable_stream(rs.stream.depth, rs.format.z16, 30)
```

Comment:
The old version was the one that was (stream_type,stream_index=-1), right? This is another version of enable_stream, right? Is it equivalent? Just want to make sure...

Reply:
Verified
![image](https://github.com/IntelRealSense/librealsense/assets/64067618/253fe5af-b172-4927-b46b-ee3609a53c1d)

I wonder if we should add a python only 1 argument function.
stream type only.
Normally the user doesn't care about stream index.
Thoughts?

Reply:
?

Reply:
It's an option, if we want better backwards compatibility. (and usage, I guess)
Should I wait before approving, then?

Reply:
I added it